src/types.ts

```diff
@@ -1,9 +1,14 @@
 export type MeasureStartMessage = {
   msgType: 'measureStart';
-  width: number;
-  staves: number;
-  clefs: string[];
-  time: string;
+  width?: number;
+  staves?: number;
+};
+
+export type AttributesMessage = {
```

Comment:
nit: This is a very generic name. Would you add a comment referencing https://www.w3.org/2021/06/musicxml40/musicxml-reference/elements/attributes/ please?

Reply:
Done